pkg/scannerv4/mappers/mappers.go

```diff
@@ -886,6 +987,10 @@ func nvdCVSS(v *nvdschema.CVEAPIJSON20CVEItem) (*v4.VulnerabilityReport_Vulnerab
 // vulnerabilityName searches the best known candidate for the vulnerability name
 // in the vulnerability details. It works by matching data against well-known
 // name patterns, and defaults to the original name if nothing is found.
+//
+// TODO: This is modified in scanner/enricher/csaf/csaf.go to prevent circular dependencies.
```

Comment:
Not sure what this comment is referencing to.

Reply:
removed this because I was able to resolve the circular dependencies